### ISSUE TITLE
tmux-fingers not integrating properly with tmux-yank

### ISSUE THREAD

**Issue by jmdaly** [Morantron/tmux-fingers#10]
Hi there,

First of all, thanks for the great plugin! I just discovered it yesterday, and am finding it really useful.

I'm having an issue with the tmux-yank integration. tmux-yank works correctly on my system, but tmux-fingers doesn't seem to be making use of it. I believe the problem is related to [this line](https://github.com/Morantron/tmux-fingers/blob/master/scripts/actions.sh#L52) in `actions.sh`. On my computer, tmux-yank sets up two bind-keys for yank:

```
$ tmux list-keys | grep tmux-yank
bind-key    -T prefix Y                run-shell -b /home/jdaly/.tmux/plugins/tmux-yank/scripts/copy_pane_pwd.sh
bind-key    -T prefix y                run-shell -b /home/jdaly/.tmux/plugins/tmux-yank/scripts/copy_line.sh
```

As a result, `$HAS_TMUX_YANK` is set to 2. The line I linked above in `actions.sh` checks if `$HAS_TMUX_YANK` is equal to 1, which is false. If I change that line to this:

```
if [ "$HAS_TMUX_YANK" != "0" ]; then
```

then I get correct integration with tmux-yank. If this is the correct fix, it would be great to have it integrated into tmux-fingers.

Thanks for your help,
John

**Comment by Morantron** [comment]
Thanks a lot for reporting and for all the detective work! :grin: 

The issue it's already fixed in 0.1.4 version. You can upgrade by:

* Pressing `prefix + U` if you are using tmux plugin manager.
* Going to tmux-fingers folder and running `git pull`.

Thanks again! I hope you are liking the plugin! :tada:

**Comment by jmdaly** [comment]
Great, thank you for the quick response! It's working well now :)
